joboard/settings.py

```diff
@@ -38,6 +38,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'joboard_home.apps.JoboardHomeConfig',
+    'login'
```

Comment:
Please split that commit from the commit in the master (it seems that you edited the last master commit)

Reply:
Just making sure I understand - should I split the change from the last commit and move it to the other one or remove the commit from this PR?

Reply:
You need to split your first commit: 978db81089e37dc4874753b15e0786db51a31e54

Reply:
Does it look ok now?

Reply:
I think yes. 
@bardielle?

Reply:
Yes:)